apps/climatemappedafrica/src/components/HowItWorks/index.js

```diff
@@ -0,0 +1,85 @@
+import { Box, Button, Grid, Typography, useMediaQuery } from "@mui/material";
```

Comment:
Ensure the components follow the standard Folder/{index.js},{Folder.js},{Folder.test.js @kelvinkipruto